components/writers/src/main/java/org/datacleaner/extension/output/CreateExcelSpreadsheetAnalyzer.java

```diff
@@ -71,11 +74,14 @@ public void validate() {
         if (sheetName.indexOf(".") != -1) {
             throw new IllegalStateException("Sheet name cannot contain dots (.)");
         }
+
+        if (file.exists() && !overwriteFile) {
+            throw new IllegalStateException("The file already exits and the columns selected do not match");
```

Comment:
Typo: The file already EXITS :)

I would also be more explicit in the message: "The file already exists and the columns selected do not match the existing records"